pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
Does this mean that a syntax error in a plugin could potentially halt the entire bot process?